vite.config.ts

```diff
@@ -22,20 +25,31 @@ const productionConfig = defineConfig({
       formats: ['es', 'umd']
     },
     rollupOptions: {
-      external: ['vue']
+      external: ['vue'],
+      output: {
+        globals: {
+          vue: 'Vue'
+        }
+      }
     }
   }
 });
 
 const demoConfig = defineConfig({
   root: path.join(root, 'demo'),
-  plugins: [VueJSX()],
+  plugins: [
+    VueJSX(),
+    Vue(),
+    WindiCSS({
+      config: path.join(root, 'tailwind.config.ts')
```

Comment:
Windinin configi neden tailwind 🤔

Reply:
Windi hem tailwind hem windi.config.ts dosyalarını okuyabiliyor o yüzden ikisinin de ortası olsun diye yapmıştım.

Reply:
I should change it to windi :D thanks.